### PR TITLE
Fix PR#1006

### PR DESCRIPTION
**Pull request by dota17**
PR #1006  should made some change comment in the code.
and its test case need to change a bit: remove duplicate test items `list`, replaced by `object` item, the data comes from its original old commit.

### CODE REVIEW

File: include/nlohmann/detail/output/output_adapters.hpp [hunk 1]
```diff
@@ -97,7 +97,8 @@ class output_string_adapter : public output_adapter_protocol<CharType>
   private:
     StringType& str;
 };
-
+/// dump to alternative string type, as defined in basic_json template
```
Comment (nlohmann) [inline include/nlohmann/detail/output/output_adapters.hpp:100]: Please remove this comment. It's technically correct, but does not help the user or developer.

Comment (dota17) [reply include/nlohmann/detail/output/output_adapters.hpp]: Ok, fixed.

File: include/nlohmann/detail/output/output_adapters.hpp [hunk 2]
```diff
@@ -97,7 +97,6 @@ class output_string_adapter : public output_adapter_protocol<CharType>
   private:
     StringType& str;
 };
-
```
Comment (nlohmann) [inline include/nlohmann/detail/output/output_adapters.hpp:100]: Sorry to be pedantic, but can you please re-add the line?

Comment (dota17) [reply include/nlohmann/detail/output/output_adapters.hpp:100]: Oh, sorry, will do now.

**Comment by nlohmann** [comment]
Thanks!

**Pull request merged**